apps/object-detection/app/eval.py

```diff
@@ -66,12 +69,14 @@ def push_to_aperturedb(db, img_id, detections, classes, source, confidence_thres
 
         q.append(abb)
 
-    db.query(q)
+    with pool.get_connection() as db:
```

Comment:
This can be also done without all the print_last_response check with a call to execute_query of connection pool.